lib/zlib.js

```diff
@@ -54,8 +53,8 @@ const codes = {
 };
 
 const ckeys = Object.keys(codes);
-for (var ck = 0; ck < ckeys.length; ck++) {
-  var ckey = ckeys[ck];
+
+for (const ckey of ckeys) {
```

Comment:
optimized better, yes, thankfully. I'm not saying *not* to do this, we should just have a benchmark that shows minimal impact. I'm all for using `for-of` in general if it doesn't slow things down.